<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QMdiArea" name="mdiArea">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>200</width>
      <height>160</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_DocNew"/>
   <addaction name="action_DocOpen"/>
   <addaction name="action_DocSave"/>
   <addaction name="action_CloseAll"/>
   <addaction name="separator"/>
   <addaction name="action_Cut"/>
   <addaction name="action_Copy"/>
   <addaction name="action_Paste"/>
   <addaction name="action_Font"/>
   <addaction name="separator"/>
   <addaction name="action_MDI"/>
   <addaction name="action_Cascase"/>
   <addaction name="action_Tile"/>
   <addaction name="separator"/>
   <addaction name="action_Quit"/>
  </widget>
  <action name="action_DocNew">
   <property name="icon">
    <iconset resource="resourse.qrc">
     <normaloff>:/ico/menu-ico/New.png</normaloff>
     <normalon>menu-ico/New.png</normalon>:/ico/menu-ico/New.png</iconset>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
   <property name="toolTip">
    <string>新建文档窗口</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="icon">
    <iconset resource="resourse.qrc">
     <normaloff>:/ico/menu-ico/Close.png</normaloff>:/ico/menu-ico/Close.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出本系统</string>
   </property>
  </action>
  <action name="action_DocOpen">
   <property name="icon">
    <iconset resource="resourse.qrc">
     <normaloff>:/ico/menu-ico/Open.png</normaloff>:/ico/menu-ico/Open.png</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
   <property name="toolTip">
    <string>打开文档</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Font">
   <property name="icon">
    <iconset resource="resourse.qrc">
     <normaloff>:/ico/menu-ico/Pink drop.png</normaloff>:/ico/menu-ico/Pink drop.png</iconset>
   </property>
   <property name="text">
    <string>字体设置</string>
   </property>
   <property name="toolTip">
    <string>字体设置</string>
   </property>
  </action>
  <action name="action_Cut">
   <property name="icon">
    <iconset resource="resourse.qrc">
     <normaloff>:/ico/menu-ico/Cut.png</normaloff>:/ico/menu-ico/Cut.png</iconset>
   </property>
   <property name="text">
    <string>剪切</string>
   </property>
   <property name="toolTip">
    <string>剪切</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="action_Copy">
   <property name="icon">
    <iconset resource="resourse.qrc">
     <normaloff>:/ico/menu-ico/Copy.png</normaloff>:/ico/menu-ico/Copy.png</iconset>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
   <property name="toolTip">
    <string>复制</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_Paste">
   <property name="icon">
    <iconset resource="resourse.qrc">
     <normaloff>:/ico/menu-ico/Paste.png</normaloff>:/ico/menu-ico/Paste.png</iconset>
   </property>
   <property name="text">
    <string>粘贴</string>
   </property>
   <property name="toolTip">
    <string>粘贴</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="action_MDI">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resourse.qrc">
     <normaloff>:/ico/menu-ico/About.png</normaloff>:/ico/menu-ico/About.png</iconset>
   </property>
   <property name="text">
    <string>MDI模式</string>
   </property>
   <property name="toolTip">
    <string>窗口式或页面模式</string>
   </property>
  </action>
  <action name="action_Cascase">
   <property name="icon">
    <iconset resource="resourse.qrc">
     <normaloff>:/ico/menu-ico/About.png</normaloff>:/ico/menu-ico/About.png</iconset>
   </property>
   <property name="text">
    <string>级联展开</string>
   </property>
   <property name="toolTip">
    <string>窗口级联展开</string>
   </property>
  </action>
  <action name="action_Tile">
   <property name="icon">
    <iconset resource="resourse.qrc">
     <normaloff>:/ico/menu-ico/About.png</normaloff>:/ico/menu-ico/About.png</iconset>
   </property>
   <property name="text">
    <string>平铺展开</string>
   </property>
   <property name="toolTip">
    <string>窗口平铺展开</string>
   </property>
  </action>
  <action name="action_CloseAll">
   <property name="icon">
    <iconset resource="resourse.qrc">
     <normaloff>:/ico/menu-ico/Check.png</normaloff>:/ico/menu-ico/Check.png</iconset>
   </property>
   <property name="text">
    <string>关闭全部</string>
   </property>
   <property name="toolTip">
    <string>关闭全部窗口</string>
   </property>
  </action>
  <action name="action_DocSave">
   <property name="icon">
    <iconset resource="resourse.qrc">
     <normaloff>:/ico/menu-ico/Save.png</normaloff>:/ico/menu-ico/Save.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="toolTip">
    <string>保存修改</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resourse.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_Quit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
